doc/releases/quincy.rst

```diff
@@ -57,7 +57,7 @@ General
 
 * telemetry: Improved the opt-in flow so that users can keep sharing the same
   data, even when new data collections are available. A new 'perf' channel that
-  collects various performance metrics is now avaiable to opt into with:
```

Comment:
I disagree with this as a feature of auto-correct. I think that "to opt into with" is sometimes appropriate, and I think that this judgment shouldn't be ceded to auto-correct.

Reply:
fwiw, this wasn't autocorrect. @anthonyeleven asked for `for` and my human inclination shifted `opt into` to `opt-in`.
https://github.com/ceph/ceph/pull/46843#discussion_r906429168

Reply:
Sorry, I messed up and missed that there were more words in the suggestion (I might have been slightly less likely to mess up if it was done in ``` suggestion form instead of free-text).